ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -15,25 +15,45 @@ Demo package contains simple demo to get and visualize result of model inference
 
 
 ## Prerequisites
-* Python 3.8+
+* [Python 3.8](https://www.python.org/downloads/)
+* [Git](https://git-scm.com/)
 
 ## Setup Demo Package
 
-1. Install Python (version 3.8 or higher), [setuptools](https://pypi.org/project/setuptools/), [wheel](https://pypi.org/project/wheel/).
+1. Install [prerequisites](#prerequisites) section. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/). For example, on Ubuntu execute the following command to get pip installed:
+   ```
+   sudo apt install python3-pip
+   ```
 
-2. Install the package in the clean environment:
-```
-python -m pip install demo_package-0.0-py3-none-any.whl
-```
+2. Create clean virtual environment:
+   ```
+   python -m pip install virtualenv
+   python -m virtualenv <directory_for_environment>
+   ```
+
+   Before starting to work inside virtual environment, it should be activated:
+   ```
+   source <directory_for_environment>/bin/activate
```

Comment:
Is it the same layout for Linux and Windows?

Reply:
Added for Windows